internal/cmd/launcher_test.go

```diff
@@ -265,6 +255,61 @@ func TestLauncherLaunch(t *testing.T) {
 	}
 }
 
+func TestStdin(t *testing.T) {
+	t.Parallel()
+
+	k6Args := []string{"k6", "archive", "-"}
+
+	ts := tests.NewGlobalTestState(t)
+	ts.CmdArgs = k6Args
+
+	// k6deps uses os package to access files. So we need to use it in the global state
+	ts.FS = afero.NewOsFs()
```

Comment:
```suggestion
	ts.FS = fsext.NewMemMapFS()
```
Is the afero dependency really needed?